config/ncg.conf

```diff
@@ -42,91 +32,77 @@ VO = dteam
   # parameters passed at invocation.
 
   <Nagios>
-    # pick global values
-    GLITE_VERSION=$GLITE_VERSION
-
-    # Directories used by NCG
-    TEMPLATES_DIR = /usr/share/grid-monitoring/config-gen/nagios
-    OUTPUT_DIR = /etc/nagios/wlcg.d
-    NRPE_OUTPUT_DIR = /etc/nagios/nrpe/
-    RESOURCES_OUTPUT_DIR = /etc/nagios
-
-    # Modify following to values used for creating MyProxy
-    # credential with command myproxy-init
-    # myproxy-init argument -k
-    MYPROXY_NAME=NagiosRetrieve
-    # myproxy-init argument -l
-    MYPROXY_USER=nagios
+    NAGIOS_ROLE = ngi
```

Comment:
explain all different values
NAGIOS_ROLE = ngi if not ngi what else can we put
when empty hosts is 1 and why?
    INCLUDE_EMPTY_HOSTS = 0
what is enable notifications? enable nagios notifications for whom
    ENABLE_NOTIFICATIONS = 0
what is SEND_TO_DASHBOARD?
    SEND_TO_DASHBOARD = 0
what is CHECK_HOSTS?
    CHECK_HOSTS = 0

    TENANT = ARGO-TEST
robot certificates. Why do we need them 
    ROBOT_CERT=/etc/nagios/globus/robocert.pem
    ROBOT_KEY=/etc/nagios/globus/robokey.pem
  when do we set   USE_ROBOT_CERT to 1
  USE_ROBOT_CERT=1
what is SEND_TO_MSG? When do we use it
    SEND_TO_MSG = 0